internal/core/software_renderer.rs

```diff
@@ -1235,6 +1240,7 @@ impl<'a, T: ProcessScene> crate::item_rendering::ItemRenderer for SceneBuilder<'
                             source_size: geometry.size,
                             format: PixelFormat::AlphaMap,
                             color,
+                            alpha: (self.current_state.alpha * 255.0) as u8,
```

Comment:
since the color already contains the alpha, this could make the application double (it's just that right now the rendered ignore the color's alpha, but it shouldn't)

Reply:
Should be better now.